src/scripts/test.js

```diff
@@ -0,0 +1,23 @@
+const list = {
```

Comment:
Your code looks good - approved

But next time delete all such test files) Also, do not forget to add DEMO LINK to the PR description